custom_components/echonetlite/const.py

```diff
@@ -1,10 +1,10 @@
 """Constants for the echonetlite integration."""
-from homeassistant.const import CONF_ICON, CONF_NAME, CONF_TYPE
+from homeassistant.const import CONF_ICON, CONF_NAME, CONF_TYPE, DEVICE_CLASS_TEMPERATURE, DEVICE_CLASS_ENERGY
+from homeassistant.components.sensor import ATTR_STATE_CLASS, STATE_CLASS_MEASUREMENT, STATE_CLASS_TOTAL_INCREASING
 from pychonet.HomeAirConditioner import ENL_FANSPEED, ENL_AIR_VERT, ENL_AIR_HORZ, ENL_AUTO_DIRECTION, ENL_SWING_MODE, FAN_SPEED, AIRFLOW_VERT, AIRFLOW_HORIZ, AUTO_DIRECTION, SWING_MODE
 
 DOMAIN = "echonetlite"
-
-SENSOR_TYPE_TEMPERATURE = "temperature"
```

Comment:
Reused inbuilt HA constant for a temperature sensor device class DEVICE_CLASS_TEMPERATURE